### ISSUE TITLE
[Bug] Everything between %% symbols gets excluded

### ISSUE THREAD

**Issue by keerah** [oleeskild/obsidian-digital-garden#52]
Looks like everything inside a pair of `%%` symbols (in md file) gets excluded
It wouldn't be such a big problem if only the code blocks were not affected as well,
and the escape symbol is not functional inside them unfortunately, this crushes a lot of things

this piece is for instance:
```batch
for %%x in (%*) do (
   set /A argCount+=1
   set "argVec[!argCount!]=%%~x"
   set "argVn[!argCount!]=%%~nx"
```
everything after `for` and before the `~x` gets lost
and then a huge part of md is also excluded until the next pair of `%%` is found somewhere in the other code block

**Comment by oleeskild** [comment]
Right, this was recently added by this PR https://github.com/oleeskild/obsidian-digital-garden/pull/45, to remove any obsidian comments. I forgot that this would also affect code blocks. I will implement a fix. Thanks for reporting it!

**Comment by oleeskild** [comment]
I've just released version 1.12.0 which should fix this issue. Let me know if it's working as expected after updating to the latest version.
